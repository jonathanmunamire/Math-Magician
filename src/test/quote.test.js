import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/quote';

describe('test if the page NavBar renders components', () => {
  test('test if the page NavBar is rendered', () => {
    const quoteSnapShot = renderer
      .create(<Quote />)
      .toJSON();

    expect(quoteSnapShot).toMatchSnapshot();
  });
});
