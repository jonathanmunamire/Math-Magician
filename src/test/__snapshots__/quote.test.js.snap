// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test if the page NavBar renders components test if the page NavBar is rendered 1`] = `
<div
  className="quote_page"
>
  <p
    className="quote"
  >
    "As far as the laws of mathematics refer to reality, they are not certain; and as far as they are certain, they do not refer to reality". -
     
    <span
      className="author"
    >
      Albert Einstein
    </span>
  </p>
</div>
`;
